src/libPMacc/include/compileTime/AllCombinations.hpp

```diff
@@ -90,6 +92,7 @@ struct AllCombinations<T_MplSeq, T_TmpResult, false >
 
     BOOST_STATIC_CONSTEXPR uint32_t rangeVectorSize = bmpl::size<MplSeq>::value;
     typedef typename bmpl::at<MplSeq, bmpl::integral_c<uint32_t, rangeVectorSize - 1 > > ::type LastElement;
+    typedef bmpl::empty<LastElement> IsLastElementEmpty;
```

Comment:
@Heikman **Don't press merge** if the author not answered all your questions. 
reasons for not answering: 
  - ignoring the mail flood
  - no time
  - the sugar/coffee level is critical low

@ax3l @Heikman 
` bmpl::empty<LastElement>` inherit from `boos::mpl::bool_<>` therefore no `::type` is needed